src/test/java/AnalyzerTest.java

```diff
@@ -0,0 +1,42 @@
+import junit.framework.Assert;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class AnalyzerTest {
+
+    @Test
+    public void analyzeTest1() {
+        Assert.assertEquals(false,Analyzer.analyze("a(na(lyz)e"));
```

Comment:
@DmitriyLitvin please use Assert.assertFalse
```suggestion
        Assert.assertFalse(Analyzer.analyze("a(na(lyz)e"));
```